lib/saml_config.rb

```diff
@@ -2,12 +2,10 @@
 
 module Saml
   class Config
-    def initialize
-      @@settings ||= build_settings.freeze
-    end
+    attr_reader :settings
 
-    def settings
-      @@settings
+    def initialize
+      @settings ||= build_settings.freeze
```

Comment:
great suggestion! I am used to seeing constants used for this type of thing, so I think this direction sounds logical. Updated at 8489c34